Makefile

```diff
@@ -20,13 +25,18 @@ LDFLAGS +="
 
 # Build for distribution
 build:
-	@echo Default build Linux amd64
+	@echo Building Mattermost Bifrost
 	env GOOS=linux GOARCH=amd64 $(GO) build -ldflags $(LDFLAGS) -o $(APPNAME) ./cmd/$(APP)
 
 # Builds the docker image
-docker:
-	docker build -t mattermost/$(APPNAME) .
-	rm -f $(APPNAME)
+build-image:
+	@echo Building Mattermost Bifrost Docker Image
+	docker build \
+	--build-arg DOCKER_BUILD_IMAGE=$(DOCKER_BUILD_IMAGE) \
+	--build-arg DOCKER_BASE_IMAGE=$(DOCKER_BASE_IMAGE) \
+	. -f build/Dockerfile \
+	-t $(MATTERMOST_BIFROST_IMAGE) \
+	--no-cache
```

Comment:
Any reason for `--no-cache`?

Reply:
@agnivade No real big reason. It's just good practise to use when you want a clean build.

Reply:
Interesting, I would love to see some links to blogs/docs on the reasoning behind why it's a best practice. But not a blocker for this PR.